.travis.yml

```diff
@@ -4,31 +4,32 @@ language: python
 # Test on PR should be execute for all branches and forks.
 branches:
   only:
-  - master
-
-python:
-  - 2.7
-
-env:
-  - TOX_ENV=py27-test-twlatest
-  - TOX_ENV=py27-test-twtrunk
-  - TOX_ENV=py34-test-twtrunk
-  - TOX_ENV=linters
+    - master
 
 matrix:
   fast_finish: true
+  include:
+  - python: 2.7
+    env: TOX_ENV=py27-test-twlatest
+  - python: 2.7
+    env: TOX_ENV=py27-test-twtrunk
+  - python: 3.5
+    env: TOX_ENV=py35-ported
+  - python: 3.5
+    env: TOX_ENV=py35-test-twtrunk
+  - python: 2.7
+    env: TOX_ENV=linters
   allow_failures:
-    - env: TOX_ENV=py34-test-twtrunk
+    - env: TOX_ENV=py35-test-twtrunk
```

Comment:
py36 instead of py35?

Reply:
Well, in master we have tox.ini with `{py33,py34,py35}-test-{twtrunk,dev}`
This is why I focused on py35.

The main part here is `TOX_ENV=py3N-ported` which is a step toward making sure we don't regress on the tests that were already ported